### PR TITLE
Add Speaker bureau page

### PR DESCRIPTION
**Pull request by AnitaYCheng**
# Pull request summary
Add speaker bureau page, including a new image.

### CODE REVIEW

**Comment by AnitaYCheng** [comment]
New page is added and the image works! The `{{ site.baseurl }}` stuff and callout boxes are still broken, but I checked and they're also broken on the `replatform-main` branch build, so I won't worry too much about it. 🙂

File: content/pages/contact/speaker-bureau.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Speaker Bureau
+permalink: /speaker-bureau/
```
Comment (beechnut) [inline content/pages/contact/speaker-bureau.md:3]: concern (content): Are people familiar with the term "speaker bureau"? That's new to me.

change (content, suggested, non-blocking): Change the `permalink` to /speakers/

change (content, suggested, non-blocking): Change the `title` to something like "Have 18F speak at your event"

**Review by beechnut — CHANGES_REQUESTED** [review]
Hi @AnitaYCheng, I've left some technical changes and suggestions for content changes. I tried to differentiate tech changes from content by adding `(content)` anywhere it's a content change.

Let me know if you want help rebasing or doing anything else, happy to hop on a call.

**Comment by AnitaYCheng** [comment]
Update: I waited until the Eleventy replatform was done and redid this work in #4075 .

**Pull request closed without merging**